src/lib.rs

```diff
@@ -171,7 +171,15 @@ impl<T> core::panic::RefUnwindSafe for Event<T> {}
 
 impl<T> fmt::Debug for Event<T> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.write_str("Event { .. }")
+        let inner = self.inner();
+        let notified_count = unsafe { (*inner).notified.load(Ordering::Relaxed) };
+        let total_count = unsafe { (*inner).list.total_listeners() };
+
+        write!(
+            f,
+            "Event {{\n  Number of notified listeners: {:?}\n  Total number of listeners: {:?}\n}}",
+            notified_count, total_count
+        )
```

Comment:
To simplify, use `debug_struct` to construct the output instead of manually formatting like this.